### DISCUSSION TITLE
Target-specific resource logging in targets pipeline

### DISCUSSION THREAD

**Discussion by koefoeden** [wlandau/autometric#2] [General]
Hi Will!

This package looks super cool! Are you planning to add some functionality that will enable autometric::log_read() to report the target that was being run at the given timestamps, such that the appropriate amount of resources can be allocated subsequently for each target in the pipeline script?  Or perhaps there is a quite straightforward way to do it currently that I do not realize? 

I considered adding simple print statements indicating the beginning and end of each target, and somehow modifying the log_read()-function to reflect this in the table - but it would probably not be a pretty solution.

**Comment by wlandau** [comment]
Thanks, I think I can make the integration smoother with #3.

**Reply by wlandau** [reply]
As of `autometric` 0.0.5.9001 (dev), you can call `autometric::log_phase_set(phase = targets::tar_name())` at the beginning of the target and then `autometric::log_phase_reset()` at the end. I plan to add this instrumentation directly into `targets` and `crew`.

**Reply by koefoeden** [reply]
Awesome - thanks for implementing this so quickly!

**Reply by wlandau** [reply]
`crew` and `targets` now use `log_phase_set()` and `log_phase_reset()` to log the names of targets/tasks as phases.
